src/Game.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * @fileOverview Game module definition
+ */
+
+/**
+ * A game engine for tic tac toe
+ *
+ * @class Game
+ * @constructor
+ * @param {Object} player1
+ * @param {Object} player2
+ */
+var Game = function(player1, player2) {
+  /**
+   * A reference to the player1 object instance
+   *
+   * @property player1
+   * @type {Object}
+   * @default undefined
+   */
+  this.player1 = player1 || {};
+
+  /**
+   * A reference to the player2 object instance
+   *
+   * @property player2
+   * @type {Object}
+   * @default undefined
+   */
+  this.player2 = player2 || {};
+
+  /**
+   * An array representing the tic tac toe game board
+   *
+   * @property gameBoard
+   * @type {Array}
+   * @default [0,0,0,0,0,0,0,0,0]
+   */
+  this.gameBoard = [0,0,0,0,0,0,0,0,0];
+
+  /**
+   * Tracks whether the game is won by either player
+   *
+   * @property gameIsWon
+   * @type {Boolean}
+   * @default true
+   */
+  this.gameIsWon = false;
+
+  /**
+   * Tracks whether the game is a draw
+   *
+   * @property gameIsDrawn
+   * @type {Boolean}
+   * @default true
+   */
+  this.gameIsDrawn = false;
+
+  /**
+   * A reference to the PubSub object instance
+   *
+   * @property pubSub
+   * @type {Object}
+   * @default undefined
+   */
+  this.pubSub = new PubSub() || {};
+
+  this.init();
+};
+
+/**
+ * Initializes the game engine
+ *
+ * @method init
+ * @returns Game
+ */
+Game.prototype.init = function() {
+  this.assignSides()
+      .runGame(this.player1);
+
+  return this;
+};
+
+/**
+ * Assigns tic tac toe sides to the players
+ *
+ * @method assignSides
+ * @returns Game
+ * @chainable
+ */
+Game.prototype.assignSides = function() {
+  this.player1.side = 1;
```

Comment:
It is probably best to let the player objects know that their sides are being set. player1.setSide(1);